commons-mongo/src/main/scala/com/avsystem/commons/mongo/core/ops/BaseSorting.scala

```diff
@@ -0,0 +1,11 @@
+package com.avsystem.commons
+package mongo.core.ops
+
+import com.mongodb.client.model.Sorts
+import org.bson.conversions.Bson
+
+trait BaseSorting extends Any with KeyHandling {
+  def ascending(): Bson = Sorts.ascending(key)
```

Comment:
Are these parens needed?

Reply:
Nope, not needed at all. Added them for consistency with other no-arg helper methods, will be nicer to use without parens though.